clr-k8s-examples/admit-kata/deploy/webhook.yaml

```diff
@@ -1,3 +1,7 @@
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
```

Comment:
As you have moved the path of the yaml files into the `deploy` subdir, do you also need to update the [README](https://github.com/clearlinux/cloud-native-setup/tree/master/clr-k8s-examples#making-kata-the-default-runtime) section so the `kubectl apply -f` points to the right place? afaict, `apply -f` is not recursive by default.

Reply:
Thanks for the catch. Will probably remove most of the section in top level readme and put a pointer to this readme